src/core/QueryManager.ts

```diff
@@ -1529,7 +1532,7 @@ export class QueryManager {
     if (include) {
       this.getObservableQueries(include).forEach((oq, queryId) => {
```

Comment:
```suggestion
      this.getObservableQueries(include).forEach((oq) => {
```

TIL that `Set.prototype.forEach` passes the "key" as the 2nd arg, which is always the same as `value` (the 1st arg). Regardless, looks like this isn't used anymore so this can also be removed.